### PR TITLE
Style error button

### PR DESCRIPTION
**Pull request by biocom**
Addresses https://github.com/gnosis/gp-swap-ui/issues/458 

Screen Shot 2021-05-03 at 12 32 08

https://user-images.githubusercontent.com/31534717/116866458-ba96e100-ac0b-11eb-82f3-92b03f1b7fe6.mov

Screen Shot 2021-05-03 at 12 23 25

### CODE REVIEW

File: src/custom/components/Button/index.tsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ import {
   ButtonOutlined as ButtonOutlinedMod,
   ButtonEmpty as ButtonEmptyMod,
   ButtonWhite as ButtonWhiteMod,
-  ButtonConfirmedStyle as ButtonConfirmedStyleMod,
-  ButtonErrorStyle as ButtonErrorStyleMod
+  ButtonConfirmedStyle as ButtonConfirmedStyleMod
+  // ButtonErrorStyle as ButtonErrorStyleMod
```
Comment (alfetopito) [inline src/custom/components/Button/index.tsx:19]: Since this is not the Mod file, you can remove this line

```suggestion
```

**Review by alfetopito — APPROVED** [review]
Looks good on the screenshots 

The deployed page has CoinGecko list bug, can't interact with it

If I have time today I'll merge with the the fix.

**Comment by biocom** [comment]
@alfetopito Yeah funny thing yesterday, when I was working on this, suddenly it felt like an infinite loop popped up. Didn't know what I did wrong. Later it made sense :)

**Review by anxolin — APPROVED** [review]

**Review by MareenG — COMMENTED** [review]
Looks great.  I was wondering if the confirmation modals should also get a design make over to fit the all over design 

Screen Shot 2021-05-04 at 10 50 14 AM

**Review by MareenG — APPROVED** [review]

**Comment by anxolin** [comment]
> Looks great. I was wondering if the confirmation modals should also get a design make over to fit the all over design

I think so @MareenG , but this is a bit more involved (and not many users use it). Could you do an issue for that?

**Comment by biocom** [comment]
@alfetopito Removed the commented out code. I guess we need the fix to develop so we can fully test before merge?

**Comment by alfetopito** [comment]
> @alfetopito Removed the commented out code. I guess we need the fix to develop so we can fully test before merge?

Updated the branch with latest from `develop`.
Should have the bug fixed now.
Anyway, Coingecko also fixed their list by now, should not be a problem anymore.

**Comment by biocom** [comment]
Thank you @alfetopito !

**Pull request merged**